routes/api.php

```diff
@@ -21,18 +21,32 @@
 /**
  * Get tokes for provided user id
  */
-Route::get('tokens/{id}', function ($id, Request $request) {
-    $userModel = get_class($request->user());
+Route::get('tokens/{resourceName}/{id}', function (
+    $resourceName,
+    $id,
+    Request $request
+) {
+    $resource = \Laravel\Nova\Fields\ResourceRelationshipGuesser::guessResource(
```

Comment:
well this is a handy method i didn't know they had.

Reply:
Found this while digging through Nova's source. I had a feeling they would have something like this because they use the "resourceName" everywhere :)